src/benchmark.cc

```diff
@@ -145,6 +145,7 @@ class ThreadManager {
     double real_time_used = 0;
     double cpu_time_used = 0;
     double manual_time_used = 0;
+    int64_t iterations = 0;
```

Comment:
size_t is the natural type to provide in the API, but I think int64_t is more appropriate for the internals because we do arithmetic with these values. In 32-bit applications, size_t could overflow and wrap*, giving wrong but potentially plausible benchmark readouts. I think it's the same reason bytes_processed and items_processed are stored as int64_t's.

* kMaxIterations seems to be 1 billion now, so > 4 threads could do it. Or in case kMaxIterations is increased in the future.